components/Auth/Login.tsx

```
@@ -50,25 +52,50 @@ const Login: React.FC<Props> = ({ navigation }) => {
   });
 
   const submitHandler = (values: values) => {
-    let self = this;
+
     axios({
       method: "POST",
-      url: "/login",
-      data: values,
+      url: "/ping",
     })
+      .then((res) => {
+
+        setIsLoading(true);
+        setIsLoadingMessage("Sending Data");
+      })
+      .then((res) => {
+
+        setTimeout(() => { }, 3000);
+
+        return axios({
+          method: "POST",
+          url: "/login",
+          data: values,
+        })
+
+      })
       .then((res) => {
         const jwt = res.data.jwt;
         console.log(res);
         console.log(jwt);
 
-        AsyncStorage.setItem("jwt", jwt);
-        if (res && res.data) {
+        return AsyncStorage.setItem("jwt", jwt);
+      })
+      .then((res) => {
+
+        setIsLoadingMessage("Login Done");
+
+        setTimeout(() => {
+          setIsLoading(false);
+          setIsLoadingMessage("");
           setIsAuth(true);
           navigation.navigate("Root");
```

Ini jangan pake hardcoded value juga yang Root, ini calon2 error jadinya kayak pattern2 yang gua kasih dibawah aja ditaruh di object atau enum